pyproject.toml

```diff
@@ -61,6 +61,7 @@ fprime-cli = "fprime_gds.executables.fprime_cli:main"
 fprime-seqgen = "fprime_gds.common.tools.seqgen:main"
 fprime-dp-write = "fprime_gds.executables.data_product_writer:main"
 fprime-gds = "fprime_gds.executables.run_deployment:main"
+fprime-seq = "fprime_gds.common.sequence.compiler:main"
```

Comment:
Should we add `fpy` to this name to distinguish it? The old `fprime-seqgen` will lose to this in tab-complete and won't process correctly.

Reply:
Ah good point. Sure, if you think that's the best idea, we can make a generic "fpy" cmdline tool whose default behavior can be compiling.